index.html

```diff
@@ -255,10 +266,10 @@ <h2>Contact me</h2>
         <input type="email" name="email" id="email" placeholder="email" required>
         <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here..."
           maxlength="500" required></textarea>
-        <input class="efecbot" type="submit" placeholder="Get in in touch">
+        <input class="efecbot" type="submit" value="Get in touch">
       </form>
     </div>
```

Comment:
- [ ] I would suggest you please remove the border of Get in touch to match the Figma design. Also provide the border-radius to match the design
### Your design
![his](https://user-images.githubusercontent.com/81727988/236277468-e603cb6a-ce3c-4ba1-bc69-f0dec533ff3e.png)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/236277495-3a09443f-2b21-4b90-8d0b-04647d57a64b.png)